virttest/qemu_vm.py

```diff
@@ -3158,8 +3158,8 @@ def __iothread_conflict_check(params):
             add_qemu_option(devices, "msg", [attr_info])
         if params.get("realtime_mlock"):
             if devices.has_option("overcommit"):
-                attr_info = ["mem-lock", params["realtime_mlock"], bool]
-                add_qemu_option(devices, "overcommit", [attr_info])
+                cmd = "-overcommit mem-lock=%s" % params["realtime_mlock"]
+                devices.insert(StrDev("overcommit", cmdline=cmd))
```

Comment:
```suggestion
                try:
                    realtime_mlock = get_boolean(params["realtime_mlock"])
                    attr_info = ["mem-lock", realtime_mlock, bool]
                except ValueError:
                    attr_info = ["mem-lock", params["realtime_mlock"], None]
                add_qemu_option(devices, "overcommit", [attr_info])
```
I think we could keep the original behavious that the bool value `realtime_mlock` can be one of either `("yes", "on", True)` or `("no", "off", False)` since user can set the value not only `on` but also `yes`. So we can support the string value type for `realtime_mlock` by specifing the vlaue type to `None`.
The above is just my opinions, pls let me know your thoughts. Thanks.

Reply:
@YongxueHong updated !